### DISCUSSION TITLE
Includes generated HTML : Code coverage results, Sphinx documentation

### DISCUSSION THREAD

**Discussion by nyue** [thegeeklab/hugo-geekdoc#114] [Q&A]
I am using Hugo with GitLab

I am also running python code coverage and sphinx documentation generation.

What is the recommended way to include/reference such data as they are already in HTML format?

**Comment by xoxys** [comment]
That's a general question and unrelated to this theme or Hugo itself. At the end there are multiple ways, and it depends on what you try to achieve. Sphinx can create standalone documentations with own themes and features. Why do you try to mix this with Hugo?

**Comment by nyue** [comment]
I was planning to get Hugo to generate a static site for a project which will
include all the release notes (generated as *.md) to be use Hugo+GeekDoc to
show up as menu on the left

I was hoping to include links to the generated Sphinx documentation and also
the unit test code coverage of the project all on a single Hugo generated SSG

Something like the following:

```
-----------------------------------
Unit test coverage results

Sphinx generated documentation

Release Notes
  2.1
  2.0
  1.0
-----------------------------------
```

I thought maybe there is some best practice of putting generated files e.g. in the static directory and some recommended way to reference them from content/*.md

No worries if the recommendation is not to mix them.

Cheers

**Comment by xoxys** [comment]
Got it. I can't give a real recommendation as I have not used it in this way. What I have already done in another project is to auto generate a python API doc as markdown files into Hugo' content directory, so Hugo can pick it up to render and build the site https://github.com/thegeeklab/corenetworks/blob/main/Makefile#L21

But I guess that's not what you are looking for.

**Comment by nyue** [comment]
@xoxys thanks for the link, even though it is not what I am after, it is great to learn from others on different approach and practices.

I will try getting our Sphinx doc steps to generate markdown files and see if that will work out for us using this sphinx 
[extension](https://pypi.org/project/sphinx-markdown-builder/)

**Reply by xoxys** [reply]
Feel free to post your setup here that you ended up with. It might be helpful for others as well :)
